### ISSUE TITLE
Mixc automatic build of simple1 project

### ISSUE THREAD

**Issue by brunobowden** [j2objc-contrib/j2objc-gradle#507]
@advayDev1 - can you modify the simple1 project so that it can be built with your `mixc-build` plugin? Since this is only a dependency for the tests, I'm more comfortable adding it in but would like to hear your thoughts as well. Given my changes to the podspecs, I'd rather add some end-to-end tests and thought this would be a better way to do it.

**Comment by advayDev1** [comment]
Yep, happy to do this.  And definitely should only be a dependency of the tests.  Our main build should be as agnostic to tools outside itself as possible.

**Comment by advayDev1** [comment]
Actually scratch that, if you first check in a pods-based xcode project for simple1, then I'll go ahead and wrap mixc-build around that.  Or did you mean for me to build `multiProject` (where you already added the appropriate xcode project) with mixc-build?
@brunobowden

**Comment by brunobowden** [comment]
@advayDev1 - over to you...
